packages/types/src/documents/app/automation.ts

```diff
@@ -61,6 +63,7 @@ export enum AutomationActionStepId {
   LOOP = "LOOP",
   COLLECT = "COLLECT",
   OPENAI = "OPENAI",
+  TRIGGER = "TRIGGER",
```

Comment:
Does this need to be updated? Should this be `TRIGGER_AUTOMATION_RUN` or something like that?